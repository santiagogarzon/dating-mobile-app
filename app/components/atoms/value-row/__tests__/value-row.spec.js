// __tests__/Intro-test.js
import React from "react";
import renderer from "react-test-renderer";
import { ValueRow } from "../index";
import * as eva from "@eva-design/eva";
import { EvaIconsPack } from "@ui-kitten/eva-icons";
import { ApplicationProvider, IconRegistry } from "@ui-kitten/components";

test("renders correctly", () => {
  const tree = renderer
    .create(
      <>
        <IconRegistry icons={EvaIconsPack} />
        <ApplicationProvider {...eva} theme={eva.light}>
          <ValueRow />
        </ApplicationProvider>
      </>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
