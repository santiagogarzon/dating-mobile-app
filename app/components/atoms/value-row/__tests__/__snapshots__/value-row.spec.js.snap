// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    appearance="default"
    level="1"
    style={
      Array [
        Object {
          "backgroundColor": "#FFFFFF",
        },
        Array [
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "space-between",
          },
          undefined,
        ],
      ]
    }
  >
    <Text
      appearance="default"
      category="s1"
      style={
        Array [
          Object {
            "color": "#222B45",
            "fontFamily": "System",
            "fontSize": 15,
            "fontWeight": "600",
          },
          undefined,
        ]
      }
    />
  </View>
  <View
    appearance="default"
    style={
      Array [
        Object {
          "backgroundColor": "#EDF1F7",
          "height": 1,
        },
        undefined,
      ]
    }
  />
</View>
`;
